packages/contracts-bedrock/test/L2/L1Block.t.sol

```diff
@@ -93,3 +113,57 @@ contract L1BlockTest is CommonTest {
         });
     }
 }
+
+contract L1BlockEcotone_Test is L1BlockTest {
+    /// @dev Sets up the test suite.
+    function setUp() public virtual override {
+        super.setUp();
+
+        bytes4 functionSignature = bytes4(keccak256("setL1BlockValuesEcotone()"));
```

Comment:
There is no need for this `setUp` function anymore

Reply:
Now that the magic values aren't used to prepopulate individual getter assertions

Reply:
removed